src/util/analyticalObject.js

```diff
@@ -126,7 +126,8 @@ export const setCurrentAnalyticalObject = async ao => {
 // Sets analytical object to open it in another app
 export const setAnalyticalObjectAndSwitchApp = async (layer, openAs) => {
     const ao = getAnalyticalObjectFromThematicLayer(layer);
-    const url = `${config.appUrl}/${APP_URLS[openAs]}/#/currentAnalyticalObject`;
+
+    const url = `${process.env.DHIS2_BASE_URL}/${APP_URLS[openAs]}/#/currentAnalyticalObject`;
```

Comment:
Suggest using hook from app-runtime instead of the process.env var (in OverlayCard where this function is called):

```
import { useConfig } from '@dhis2/app-runtime'
...
const { baseUrl } = useConfig()

setAnalyticalObjectAndSwitchApp(layer, type, baseUrl)
```

Reply:
Fixed!